apps/src/templates/manageStudents/AddMultipleStudents.jsx

```diff
@@ -0,0 +1,83 @@
+import React, {Component, PropTypes} from 'react';
+import {connect} from 'react-redux';
+import {addMultipleAddRows} from './manageStudentsRedux';
+import Button from '../Button';
+import i18n from "@cdo/locale";
+import BaseDialog from '../BaseDialog';
+import DialogFooter from "../teacherDashboard/DialogFooter";
+
+const styles = {
+  dialog: {
+    paddingLeft: 20,
+    paddingRight: 20,
+    paddingBottom: 20
+  }
+};
+
+class AddMultipleStudents extends Component {
+  static propTypes = {
+    // Provided by redux
+    addMultipleStudents: PropTypes.func.isRequired,
+  };
+
+  state = {
+    openDialog: false,
+  };
+
+  openDialog = () => {
+    this.setState({openDialog: true});
+  };
+
+  closeDialog = () => {
+    this.setState({openDialog: false});
+  };
+
+  add = () => {
+    const value = this.refs.studentsTextBox.value;
+    this.props.addMultipleStudents(value.split("\n"));
+    this.closeDialog();
+  };
+
+  render() {
+    return (
+      <div>
+        <Button
+          onClick={this.openDialog}
+          color={Button.ButtonColor.gray}
+          text={i18n.addStudentsMultiple()}
+        />
+        <BaseDialog
+          useUpdatedStyles
+          uncloseable
+          isOpen={this.state.openDialog}
+          style={styles.dialog}
+        >
+          <h2>{i18n.addStudentsMultiple()}</h2>
+          <div>
+            {i18n.addStudentsMultipleInstructions()}
+          </div>
+          <textarea name="textarea" rows="15" cols="70" ref="studentsTextBox">
```

Comment:
May not necessary to fix in this PR, especially if there will be a styling pass later, but it'd be nice to size this textbox to match the dialog width instead of using rows/cols.  Also not sure it should be resizable.

Reply:
I'll do a style pass all together 👍